tracer/test/test-applications/regression/Sandbox.ManualTracing/Program.cs

```diff
@@ -19,15 +21,14 @@ public static int Main(string[] args)
         {
             // Set the minimum permissions needed to run code in the new AppDomain
             PermissionSet permSet = new PermissionSet(PermissionState.None);
-            permSet.AddPermission(new SecurityPermission(SecurityPermissionFlag.Execution)); // Necessary to run code.
-            // permSet.AddPermission(new WebPermission(PermissionState.Unrestricted)); // Necessary to emit traces. If commented out, the Tracer will not send Traces but it should not crash.
-            // permSet.AddPermission(new FileIOPermission(PermissionState.Unrestricted)); // Necessary to access the file system.
+            permSet.AddPermission(new SecurityPermission(SecurityPermissionFlag.Execution)); // REQUIRED to run code.
+            // permSet.AddPermission(new WebPermission(PermissionState.Unrestricted)); // REQUIRED for application to send traces to the Agent over HTTP.
```

Comment:
Mirroring @andrewlock's comment in `Sandbox.AutomaticInstrumentation/Program.cs`: should this be uncommented if it's required?

Reply:
So on this one, both ways would be useful:

1. Comment it out and use the test to make sure that we don't crash hard when we don't have access to the network
2. Uncomment it so that we have access to the network and see that we have traces being submitted end-to-end

The first option would be nice since it's the most restrictive environment, but the second option is also good because we can see end-to-end traces. Which would you prefer? Maybe later we could do approach number 1 if we created a console exporter to validate that traces were created/exported

Reply:
I misunderstood the comment, thanks! This seems good now now.